realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1448,33 +1449,32 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeSetPrimaryKey(
         const char* table_name = table->get_name().data();
         size_t row_index = pk_table->find_first_string(io_realm_internal_Table_PRIMARY_KEY_CLASS_COLUMN_INDEX, table_name);
 
-        // I
         if (columnName == NULL || env->GetStringLength(columnName) == 0) {
-            // No primary key set. Remove any previous set keys
+            // No primary key set => Remove any previous set keys
             if (row_index != realm::not_found) {
                 pk_table->remove(row_index);
             }
             return jlong(io_realm_internal_Table_NO_PRIMARY_KEY);
         }
         else {
-            JStringAccessor columnName2(env, columnName);
-            size_t primary_key_column_index = table->get_column_index(columnName2);
+            JStringAccessor NewPrimaryKeyColumn(env, columnName);
```

Comment:
Respect case convention in C++: lowercase with underscores: not "NewPrimaryKeyColumn"